app/models/reminder_mailing_volunteer.rb

```diff
@@ -0,0 +1,32 @@
+class ReminderMailingVolunteer < ApplicationRecord
+  attr_accessor :selected
+
+  before_create :remove_or_process_email_content
+
+  belongs_to :reminder_mailing
+  belongs_to :volunteer
+  belongs_to :reminder_mailable, polymorphic: true, optional: true
+
+  private
+
+  def remove_or_process_email_content
+    return destroy if selected && selected.to_i.zero?
+    self.body = reminder_mailing.body % template_variables
+    self.subject = reminder_mailing.subject % template_variables
```

Comment:
Not sure if it makes sense to pre-render these fields and store them in the DB, what happens if one of the template variables change until the mails are sent? Another option would be to render them directly when the actual mails are sent, but it looks like that's not implemented yet.

Reply:
your right. True. I just thought I do it here to fill them as a property. But maybe your right.
But it is not much of a problem tho. I can just convert this into a method, that will spit the substituded subject and body out.